lib/tasks_private/add_user.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# This script adds a user to the Azure dev environment using their company
+# userid. By default, it will create a resource group in the east US region
```

Comment:
I think company userid could just be replaced by userid, it could easily be a personal account

Reply:
Updated.